### PR TITLE
Moved volume control to VS1053 class, code cleanup.

### PR DESCRIPTION
**Pull request by johntumath**

### CODE REVIEW

File: L5_Application/VS1053.cpp [hunk 1]
```diff
@@ -105,66 +103,21 @@ void VS1053::sineTest(uint8_t n, uint16_t ms) {
...
 }
 
-void VS1053::setVolume(uint8_t left, uint8_t right) {
-  uint16_t volume;
-  volume = left;
-  volume <<= 8;
-  volume |= right;
-  sciWrite(SCI_VOL, volume);
+void VS1053::sendVolume(uint8_t left, uint8_t right) {
+  uint16_t v;
```
Comment (ColePBryan) [inline L5_Application/VS1053.cpp]: Poor variable naming, please change before pr is accepted

File: L5_Application/VS1053.cpp [hunk 2]
```diff
@@ -105,66 +103,21 @@ void VS1053::sineTest(uint8_t n, uint16_t ms) {
...
+  v = left;
+  v <<= 8;
+  v |= right;
+  sciWrite(SCI_VOL, v);
+}
+void VS1053::setVolume(uint8_t v)
```
Comment (ColePBryan) [inline L5_Application/VS1053.cpp]: Poor variable naming, please change before pr is accepted

File: L5_Application/VS1053.h [hunk 3]
```diff
@@ -29,18 +29,16 @@ class VS1053 {
...
     virtual ~VS1053();
     VS1053();
     GPIO* DREQ;
     GPIO* _CS;
     GPIO* _DCS;
     LabSpi0 SPI0;
-    FIL currentTrack;
-    bool playingMusic;
-    uint8_t musicBuffer[512];
+    uint8_t volume;
```
Comment (ColePBryan) [inline L5_Application/VS1053.h]: Great variable name!

File: L5_Application/VS1053.cpp (hunk 1 above)
Comment (johntumath) [reply L5_Application/VS1053.cpp]: Changes made per request.

File: L5_Application/VS1053.cpp (hunk 2 above)
Comment (johntumath) [reply L5_Application/VS1053.cpp]: Changes added in bdf5957

**Pull request merged**